docs/source/posts/2024/2024-05-28-week-0-robin.rst

```diff
@@ -0,0 +1,89 @@
+Week 0: Community Bonding!
+==========================
+
+.. post:: May 29 2024
+   :author: Robin Roy
+   :tags: google
+   :category: gsoc
+
+Hi, I'm Robin and I'm a 2nd year CS undergrad from Vellore Institute of Technology, Chennai. During GSoC '24 my work will be to build an LLM chatbot which will help the community by answering their questions.
+Scientific visualization is often complicated and hard for people to get used to - "Although 3D visualization technologies are advancing quickly, their sophistication and focus on non-scientific domains makes it hard for researchers to use
+them. In other words, most of the existing 3D visualization and computing APIs are low-level
+(close to the hardware) and made for professional specialist developers." [`FURY-Statement of Need <https://joss.theoj.org/papers/10.21105/joss.03384>`_]. FURY is our effort to bridge this gap with an easy-to-use API. With LLMs, the goal is to take this one step further and make it even simpler for people to get started. By reducing the barrier to entry, we can bring more people into this domain. Visualization should not be the most time-consuming thing for an engineer/researcher, it is supposed to just happen and help them accelerate faster.
+
+My Community Bonding Work
+--------------------------
```

Comment:
Remove one separator, separator line should be the same length as the title length